Mathlib/Data/PFunctor/Univariate/Basic.lean

```diff
@@ -62,22 +62,29 @@ instance Obj.inhabited [Inhabited P.A] [Inhabited α] : Inhabited (P α) :=
 
 instance : Functor.{v, max u v} P.Obj where map := @map P
 
-protected theorem map_eq {α β : Type u} (f : α → β) (a : P.A) (g : P.B a → α) :
-    @Functor.map P.Obj _ _ _ f ⟨a, g⟩ = ⟨a, f ∘ g⟩ :=
+@[simp]
+theorem map_eq_map {α β : Type v} (f : α → β) (x : P α) : f <$> x = P.map f x :=
+  rfl
```

Comment:
Can you add a docstring to this theorem explaining why it is simp (mentioning that we prefer `PFunctor.map` due to universe polymorphism)